webapp/src/utils/adapters/TSMessageAdapter.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { MessageData } from '.';
+import { type MessageData } from '.';
```

Comment:
Why are these changes required?

Reply:
I have a problem in the build somewhere and this fix it (can not remember now where)